languages/csharp/AppEncryption/README.md

```diff
@@ -197,22 +71,19 @@ AWSConfigs.AWSRegion = "us-west-2";
 
 // Build the DynamoDB Metastore.
 IMetastorePersistence<JObject> dynamoDbMetastorePersistence = DynamoDbMetastorePersistenceImpl.NewBuilder().Build();
+```
 
-// Use the Metastore for the session factory
-using (AppEncryptionSessionFactory appEncryptionSessionFactory = AppEncryptionSessionFactory.NewBuilder("productId", "systemId")
-     .WithMetastorePersistence(dynamoDbMetastorePersistence)
-     .WithCryptoPolicy(policy)
-     .WithKeyManagementService(keyManagementService)
-     .Build()) {
+#### In-memory Metastore (FOR TESTING ONLY)
 
-    // ...
-}
+```c#
+MetastorePersistence<JObject> metastorePersistence = new MemoryPersistenceImpl<>();
 ```
 
-### Key Management Service
-Please refer to the [Java lib's notes](../../java/app-encryption#key-management-service) until documentation cleanup/refactor is complete. Until then, we will simply provide code usage examples here.
+### Define the Key Management Service
+Detailed information about the Key Management Service can be found [here](../../../docs/KeyManagementService.md).
```

Comment:
nit: can we use `Key Management Service` (with spaces) in the java readme as well? (may need to merge master into this branch)